src/content/en/fundamentals/instant-and-offline/get-started/httprequests-5.md

```diff
@@ -0,0 +1,265 @@
+
+Although this technique can be applied to any group of images, it is most frequently used 
+with small images such as icons, where extra HTTP requests to fetch multiple small graphics 
+are particularly wasteful.
+
+The basic idea is to combine small images into one physical image file and then use CSS 
+background positioning to display the right part of the image at the right place on the 
+page. The CSS repositioning is fast and seamless, works on an already-downloaded resource, 
+and makes an excellent tradeoff for the otherwise-required multiple HTTP requests and image 
+downloads.
+
+For example, you might have a series of social media icons with links to their respective 
+sites or apps. Rather than downloading three (or perhaps many more) individual images, you 
+might combine them into one image file, like this.
+
+![socialmediaicons.png](images/image_500.png)
+
+Then, instead of using different images for the links, just retrieve the entire image once 
+and use CSS background positioning for each link to display the correct part of the image 
+for the link. 
+
+Here's some sample CSS.
+
+```
+a.facebook {
+   display: inline-block;
+   width: 64px; height: 64px;
+   background-image: url("socialmediaicons.png");
+   background-position: 0px 0px;
+   }
+a.twitter {
+   display: inline-block;
+   width: 64px; height: 64px;
+   background-image: url("socialmediaicons.png");
+   background-position: -64px 0px;
+   }
+a.pinterest {
+   display: inline-block;
+   width: 64px; height: 64px;
+   background-image: url("socialmediaicons.png");
+   background-position: -128px 0px;
+   }
+```
+
+Note the extraneous `background-position` property in the `facebook` class. It's not 
+necessary, as the default position is 0,0 but is included here for consistency. The other 
+two classes simply shift the image left horizontally relative to its container by 64px and 
+128px, respectively, leaving the appropriate image section visible in the 64-by-64 pixel 
+link "window".
+
+Here's some HTML to use with that CSS.
+
+```
+<p>Find us on:</p>
+<p><a class="facebook" href="www.facebook.com"></a></p>
+<p><a class="twitter" href="www.twitter.com"></a></p>
+<p><a class="pinterest" href="www.pinterest.com"></a></p>
+```
+
+Instead of including separate images in the links themselves, just apply the CSS classes 
+and leave the link content empty. This simple technique saves you two HTTP requests by 
+letting CSS do the image shifting behind the scenes. If you have lots of small images -- 
+navigation icons or function buttons, for example -- it could save you many, many trips 
+to the server.
+
+You can find a brief but excellent article about this technique, including working examples, 
+at [WellStyled](http://wellstyled.com/css-nopreload-rollovers.html).
+
+## A Caveat
+
+In our discussion of combining text and graphics, we should note that the 
+newer HTTP/2 protocol may change how you consider combining resources. For example, common and 
```

Comment:
Please link, wherever possible, to existing guidance on /web related to the topics being written about the first time they come up. "HTTP/2" should be a link to https://developers.google.com/web/fundamentals/performance/http2/. You reference this article lower down, but linking to it inline makes it easier to just dive into that doc when you discover a term you don't understand.